### DISCUSSION TITLE
More Versions (UNHASHED)

### DISCUSSION THREAD

**Discussion by nasko222** [nasko222/the-minecraft-recreation-project#160] [Ideas]
Here are some trees to older points of the repo. I'll update these with stuff I find interesting. The versions here are unhashed and if you're a verifier you need to hash them in MD5. But I assume you're a regular user so just ignore that.

* Beta 1.8 Devs (Pre-BlueStaggo's builds) (AND ALSO CURRENTLY LATEST B1.8 DEVS) - https://github.com/nasko222/the-minecraft-recreation-project/tree/51784a4a25b14950883dfd6efeedaff0e1de3856

* Pre-Classic Speculations (THESE ARE COMPLETELY MADE UP AND ARE JUST AS AN IDEA OF PRECLASSIC PROGRESSION, DONT TAKE IT SERIOUSLY) - https://github.com/nasko222/the-minecraft-recreation-project/tree/794fe4bf688dc21a25a1fa806100f6689d5ccbd7

* 0.24 With saving (0.30 based 0.24 dev builds that have saving because 0.30 saves properly, THEY ARE INACURRATE) - https://github.com/nasko222/the-minecraft-recreation-project/tree/79b564ecb17d1f29227cb61de88035815079c3c3

* The very first upload on Github (Remakes 2021, most of them are very inaccurate haha) - https://github.com/nasko222/the-minecraft-recreation-project/tree/6c1bca53fc9ab3330249f0f1190f28161590b86a

* Method's original public remakes. HERE IN THIS THREAD. Posted by minecraft12 - https://github.com/nasko222/the-minecraft-recreation-project/discussions/160#discussioncomment-7734041 (WARNING, SOME OF THEM MIGHT REQUIRE JAVA 16+ TO RUN)

Tell me if you are interested in other old and significant snapshots of the repo.

# Update: For developers who want to post their recreations move to https://github.com/nasko222/the-minecraft-recreation-project/discussions/170 instead. This discussion is only for recreations that were ever part of TMRP.

**Comment by G23M08** [comment]
The pre classic speculations is what that guy is using to post

**Reply by nasko222** [reply]
Which guy?

**Reply by G23M08** [reply]
A que that used these versions saying that you and method made them and 0.0.11_1 and 0.0.14 Herobrine test (I hate these names)

**Reply by nasko222** [reply]
Nope these never existed. I think you got the wrong idea. This post is for scavenging older versions of remakes or misc or the speculation remakes that got deleted. Herobrine test sounds unreal and it never existed here in TMRP

**Reply by G23M08** [reply]
I know he used your code to make these versions but he used the fake preclasics also if you go on YouTube and search "Minecraft pc-10" you will find it

**Reply by G23M08** [reply]
It's Tristan something

**Reply by nasko222** [reply]
If you talk about Tristan Corso this dude is completely unhinged, if you see his vlog vids he is only making weird noises and talking complete random words. Its horrifying to some extent. Also he stole my code without permission and then started to make weird versions like "11a_03-dev" 64x64x64 or some shit, and pings me and method for making these, we never even thought about stuff like that so basically don't trust anything he says. One of his descriptions were "She is minecraft, she is coded on nasko222 computer by method"

**Reply by nasko222** [reply]
I've also spoken to him and his parents in VC and theyre similar, they barely understand the shit I am saying and tristan repeats random words. i have a video of him screaming and repeating "REUPLOAD!"

But overall he has been obsessive around me and method, probably because he likes the project.

**Reply by G23M08** [reply]
I will say a cringey qute "oscetion is biggest form of flattery"

**Reply by G23M08** [reply]
also hes german

**Comment by JONESTHEBIRD** [comment]
I would be interested in seeing older remakes

**Reply by nasko222** [reply]
Will try to get methods original ones here

**Comment by minecraft12y** [comment]
[method old remakes.zip](https://github.com/nasko222/the-minecraft-recreation-project/files/13531778/method.old.remakes.zip)
did someone say method, old and remakes?

**Reply by nasko222** [reply]
hot remakes hot hot remakes

**Reply by nasko222** [reply]
nice zip file bro damn even i didnt had all of those XD

**Reply by Scorpionadi** [reply]
Minecraft12y do you have the original remakes by tolerable druid?

**Reply by minecraft12y** [reply]
Nope, there's no other remakes I have.

**Comment by nasko222** [comment]
updated few links here, 0.24 with saving, 2021 remakes (first tmrp github upload), and methods PUBLIC 2019-2020 remakes (archived by minecraft 12)

**Comment by Illia24** [comment]
Wow its incredible that you guys saved old remakes good job!

**Comment by G23M08** [comment]
indev 20091231dev the tnt works but indev 20091231-2042 the tnt dosent work

**Comment by nasko222** [comment]
send log

**Reply by G23M08** [reply]
when the tnt blows up in the dev one it dosent send a log

**Reply by nasko222** [reply]
oh i thought the game crashes, yea in the dev the tnt behaves like that.

**Reply by G23M08** [reply]
oh k

**Comment by 483aa** [comment]


**Reply by nasko222** [reply]
Please clarify in your post that your software is NOT associated with The Minecraft Recreation Project.

**Reply by nasko222** [reply]
@CreepyTDYA Hi, I removed your mediafire link and created new discussion for everyone elses recreations that are not part of TMRP.

Feel free you and everyone else to post it there. https://github.com/nasko222/the-minecraft-recreation-project/discussions/170

**Comment by nasko222** [comment]
@CreepyTDYA  Hi, I removed your mediafire link and created new discussion for everyone elses recreations that are not part of TMRP.

Feel free you and everyone else to post it there. https://github.com/nasko222/the-minecraft-recreation-project/discussions/170

**Reply by 483aa** [reply]
Okay

**Comment by nasko222** [comment]
@CreepyTDYA Reminder to post your recreations in the appropriate discussion, thanks
